packages/code-du-travail-frontend/src/search/search.service.js

```diff
@@ -18,13 +18,22 @@ const fetchResults = endpoint => (query = "", excludeSources = "") => {
   });
 };
 const searchResults = fetchResults("search");
-const suggestResults = fetchResults("suggest");
 
-const suggestMin = (query, excludeSources) => {
+const suggestResults = query => {
+  const url = `${SUGGEST_URL}?q=${query}`;
+  return fetch(url).then(response => {
+    if (response.ok) {
+      return response.json();
+    }
+    throw new Error("suggester: Un problème est survenu.");
+  });
+};
+
+const suggestMin = query => {
```

Comment:
Simplified to 

```js
const suggestMin = query => query.length > 2 
  ? suggestResults(query)
  : Promise.resolve([]);
```

Reply:
oki mais j'ai suis pas certain que ça simplifie la lecture ;)